frame/system/src/weights.rs

```diff
@@ -56,80 +57,80 @@ pub trait WeightInfo {
 /// Weights for frame_system using the Substrate node and recommended hardware.
 pub struct SubstrateWeight<T>(PhantomData<T>);
 impl<T: crate::Config> WeightInfo for SubstrateWeight<T> {
-	fn remark(b: u32, ) -> Weight {
-		(574_000 as Weight)
-			// Standard Error: 0
-			.saturating_add((1_000 as Weight).saturating_mul(b as Weight))
+	fn remark(_b: u32, ) -> Weight {
+		(0 as Weight)
```

Comment:
has it been resolved?
the optimizer threw it away in the benchmark as unused?

Reply:
yeah i think it is okay here. per byte fee and base extrinsic weight should be the right economic properties for this call